### PR TITLE
GEOMESA-1157 Add commons-logging and guava to install script

### PR DESCRIPTION
**Pull request by jahhulbert-ccri**
* These are needed for command line tools too work (e.g. ingest)

Signed-off-by: Andrew Hulbert <andrew.hulbert@ccri.com>

### CODE REVIEW

File: geomesa-tools/bin/install-hadoop-accumulo.sh [hunk 1]
```diff
@@ -7,6 +7,21 @@
...
+# geomesa tools lib dir or the WEB-INF/lib dir of geoserver.
+
+accumulo_version="${accumulo.version}"
+hadoop_version="${hadoop.version}"
+zookeeper_version="${zookeeper.version}"
+thrift_version="${thrift.version}"
+
+# for hadoop 2.5 and 2.6 to work we need these
+guava_version="11.0.2"
+com_log_version="1.1.3"
```
Comment (elahrvivaz) [inline geomesa-tools/bin/install-hadoop-accumulo.sh]: will this still work with older versions of hadoop?

Comment (jahhulbert-ccri) [inline geomesa-tools/bin/install-hadoop-accumulo.sh]: 2.4 uses the same guava/commons as well...anything further back than that i think we'd have issues with the CL tools with Classpath anyways.

Comment (elahrvivaz) [inline geomesa-tools/bin/install-hadoop-accumulo.sh]: sounds good - merging

**Comment by elahrvivaz** [comment]
885d5b60ea5832b1a2a902c3f57d2e0b60e9d8f6

**Pull request closed without merging**